src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Tasks/GetDotNetHost.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.InteropServices;
+using System.IO;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Microsoft.Extensions.CommandLineUtils;
+
+namespace Microsoft.AspNetCore.BuildTools
+{
+    public class GetDotNetHost : Task
```

Comment:
I'm shocked there isn't an already present MSBuid property containing this value.